tomee/content.md

```diff
@@ -0,0 +1,58 @@
+#What is TomEE?
+
+Apache TomEE, pronounced "Tommy", is an all-Apache Java EE Web Profile certified stack where Apache Tomcat is top dog. Apache TomEE is assembled from a vanilla Apache Tomcat zip file. We start with Apache Tomcat, add our jars and zip up the rest. The result is Tomcat with added EE features - TomEE.
+
+%%LOGO%%
+
+Versions 1.X of Apache TomEE are Java EE 6 Web Profile certified.
+
+Versions 7.X of Apache TomEE are Java EE 7 Web Profile certified.
+
+Apache TomEE comes with four different flavors, Web Profile, JAX-RS, Plus and Plume.
+
+-	Apache TomEE Web Profile delivers Servlets, JSP, JSF, JTA, JPA, CDI, Bean Validation and EJB Lite.
+-	Apache TomEE JAX-RS (RESTfull Services) delivers the Web Profile plus JAX-RS (RESTfull Services).
+-	Apache TomEE Plus delivers all that is in the Web Profile and JAX-RS (RESTfull Services), plus EJB Full, Java EE Connector Architecture, JMS (Java Message Service) and JAX-WS (Web Services).
+-	Apache TomEE Plume delivers all that is in the Plus Profile, but includes Mojarra and EclipseLink support.
+
+All Dockerfile files of supported versions can be found at https://github.com/tomitribe/docker-tomee
+
+# How to use this image.
+
+Run the default TomEE server (`CMD ["catalina.sh", "run"]`):
+
+docker run -it --rm tomee/<flavor>:<tomeeversion>\-<java-version>\-
```

Comment:
All of the example commands have image names that don't match what was proposed in the PR to ``docker-library/official-images``.  I'm seeing tag names of the form: ``tomee:8-jre-1.7.2-plus``.